ed25519.go

```diff
@@ -0,0 +1,81 @@
+package jwt
+
+import (
+	"errors"
+
+	"crypto/ed25519"
+)
+
+var (
+	ErrEd25519Verification = errors.New("ed25519: verification error")
+)
+
+// Implements the EdDSA family
+// Expects *ed25519.PrivateKey for signing and *ed25519.PublicKey for verification
```

Comment:
Minor nit pick: it actually does expect `ed25519.PrivateKey` rather than `*ed25519.PrivateKey`, if you look at the `Sign` method below.  But this seems to be the expected outcome of calling `ParsePKCS8PrivateKey`. In the ecdsa case, it contains a `*ecdsa.PublicKey`.

 I wonder, if this is an inconsistency in the Go crypto API or is `ed25519.PrivateKey` an interface rather than a struct?

Reply:
don't know, but i'm fix comments =)